docs/_static/css/chemiscope.css

```diff
@@ -14,3 +14,11 @@ pre {
 .admonition.note a:hover {
     color: #18bc9c;
 }
+
+.sidebar-logo {
+    display: block;
+    margin: 0 auto;
+    max-width: 100%;
+    clip-path: inset(9px 9px 25px 9px);
```

Comment:
It this to remove the dark blue border around the logo? I find the result a bit jarring:

<img width="1728" alt="Screenshot" src="https://github.com/user-attachments/assets/35cc643d-7e75-4896-8641-d0b6c7985396">

Maybe using a lighter color of blue for the background would blend it better? I can try to play with the colors

Reply:
so the idea is to have a single svg for the logo-cum-name and the square with just the figure. feel free to play with colors of the website, the blue border in the full logo uses the same hue in the website, so I'd keep it.

Reply:
Ok, I'll keep the same hue but move around in saturation/lightness

Reply:
I could not get the right amount of white padding around the "graph axes", so I added a separate file for it. The hack was nice, but two SVG is not what will kill this repo ^_^